packages/ui/src/types.ts

```diff
@@ -0,0 +1,3 @@
+import type { Story } from '@prezly/sdk';
+
+export type StoryWithImage = Story & Pick<Story.ExtraFields, 'thumbnail_image'>;
```

Comment:
Hey @e1himself, I wonder if we should include the `thumbnail_image` into base `Story` object at some point. We're using this field in every theme, and this type we've been using feels a bit like a hack 🤔

Reply:
>  I wonder if we should include the thumbnail_image into base Story object at some point. 

Hey Oleg! 

I'm not sure yet. The thumb rule for picking defaults for Stories API response fields is whether it is needed on the listing page (in the app), and `thumbnail_image` is not needed there.

But I see your point. I'll need to think about it.